vars/artifactSetVersion.groovy

```diff
@@ -32,7 +32,7 @@ def call(Map parameters = [:], Closure body = null) {
         def gitUtils = parameters.juStabGitUtils ?: new GitUtils()
 
         if (gitUtils.insideWorkTree()) {
-            if (sh(returnStatus: true, script: 'git diff --quiet HEAD') != 0)
+            if (! gitUtils.isWorkTreeClean())
```

Comment:
You probably want to throw some error in most cases if the workspace is dirty. Therefore, I would suggest to rename the step to `isWorkTreeDirty` and move the negation inside the function.

Reply:
changed